podpac/datalib/egi.py

```diff
@@ -229,7 +227,7 @@ def eval(self, coordinates, output=None, selector=None):
             raise e
 
         # run normal eval once self.data is prepared
-        return super(EGI, self).eval(coordinates, output)
+        return super(EGI, self).eval(coordinates, output=output, _selector=_selector)
```

Comment:
This is surprising. Should this be `super(EGI, self)._eval`?

Might be an infinite recursion here `Node.eval` -> `EGI._eval` -> `Node.eval` -> `EGI._eval` etc, but you want `Node.eval` -> `EGI._eval` -> `Datasource._eval`.

Reply:
AWESOME CATCH! I thought I got them all. Ok. I'll fix.